### PR TITLE
Add two column layout to component details

### PR DESCRIPTION
**Pull request by araiford**
This PR focuses on applying the two-column layout to the latest content for:

Page Layouts
Tabset
Alert Bar
Dropdown

Primary changes:

- **Adjusted the Header markdown hierarchy**
  - This change matches up the styles in Ty's website spec file, to the H* styles that have been documented during the surge.
  - Note that the styles we are getting from helix-ui DO NOT currently match the spec from the design surge.  Once the helix-ui code is adjusted to match design spec, the styles, like so many stars, will align
- **Added helix columns to the docs listed above.**
  - I would like feedback on the responsive classes I added to the markdown
- **Added tweaked redline specs from Tabs and Alert Bars**
  - I adjusted some of the redlines from Tabs and Alert Bars to be aligned with current standards for redlining.

Secondary Changes:

- Changes the body background to white
- Changes "figure" background color to fafafa
- These were both decided in a Helix Bowling session, and approved by Ty.

### CODE REVIEW

File: _component_design/alert-bar.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ title: Alert Bar
 parent: Notifications
 layout: component
 category: Components
-usage: |
-  These tooltips provide users with additional information while completing
-  tasks and alert them to new features.
+usage: Notification bars are best used for information that will remain static barring user intervention. Such as an Info message surrounding a     new feature available to a user, or an Error notification informing the user there credit card has been unable to be processed this month.
```
Comment (robb-romans) [inline _component_design/alert-bar.md]: Use notification bars for information that will remain static unless
changed by a user. For example, you could use notification bars for
Info messages about a new feature available to a user, or an Error
message informing the user that their credit card was unable to be
processed this month.

File: _component_design/alert-bar.md [hunk 2]
```diff
@@ -14,57 +12,42 @@ last-modified: 2017-08-17
...
-  to least severity level:
-  * Error > Warning > Success > Informational
+* Ensure notification bars have implicit priority based on the alert type from most to least severity level: Error > Warning > Success > Informational
 * Ensure notification bars are used sparingly and limited to one per page.
-  * In the event a higher priority alert bar needs to be shown on a page that
-    already contains a  notification bar. The higher level alert is placed
-    *above* the lower priority bar.
-
-# Specifications
+* In the event that multiple alert bars are present, they should be stacked in order of criticality, high-to-low, top-to-bottom.
```
Comment (robb-romans) [inline _component_design/alert-bar.md]: For multiple alert bars, stack them in order of...

File: _component_design/alert-bar.md [hunk 3]
```diff
@@ -76,45 +59,49 @@ optional:
...
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Multi-line alerts
 
-Your component has initialized, but it is empty, no data or items. Now may be a
-good time to get the user to perform an action, or to reward them.
-Ensure you include redlines for the specific changes in this state.
+Alert messages should be as concise and clear as possible. We recommend keeping to
+a limit of 140-160 characters. Following this rule will keep an alert message as a
+single-line comment (at Desktop and Wall breakpoints). However, in instances where a
+wrap is required, alert bars will following the wrapping behavior displayed.
```
Comment (robb-romans) [inline _component_design/alert-bar.md]: please rewrite "alert bars will following the wrapping behavior displayed"

File: _component_design/alert-bar.md [hunk 4]
```diff
@@ -76,45 +59,49 @@ optional:
...
 {% endfigure %}
+{% endcolumn %}
+</div>
 
-## Status types
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+#### Status types
 
 The alert bar pattern seeks to provide a user with at a glance status of a
```
Comment (robb-romans) [inline _component_design/alert-bar.md]: s/seeks to provide/provides/g

**Comment by robb-romans** [comment]
A few suggestions inline

**Comment by araiford** [comment]
Thanks so much @robb-romans 

Updates made per your suggestions.

**Pull request merged**